po/stellarium-skycultures/POTFILES.in

```diff
@@ -134,6 +134,7 @@ skycultures/seri/star_names.fab
 skycultures/russian_siberian/constellation_names.eng.fab
 skycultures/russian_siberian/star_names.fab
 skycultures/russian_siberian/planet_names.fab
+skycultures/tibetan_mansions/asterism_names.eng.fab

```

Comment:
wrong path now

Reply:
Fixed.

Reply:
Where is the fix?

Reply:
In the previous commit #7d1789f

Reply:
Commit #7d1789f has changes in the file `skycultures/po4a.config` only